tests/DoctrineORMModuleTest/Yuml/YumlControllerFactoryTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace DoctrineORMModuleTest\Yuml;
+
+use DoctrineORMModule\Yuml\YumlController;
+use DoctrineORMModule\Yuml\YumlControllerFactory;
+use Zend\ServiceManager\AbstractPluginManager;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class YumlControllerFactoryTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCreateService()
+    {
+        $config = [
+            'zenddevelopertools' => [
+                'toolbar' => [
+                    'enabled' => true
+                ]
+            ]
+        ];
+
+        $serviceLocator = $this->getMockBuilder(ServiceLocatorInterface::class)
+            ->getMock();
```

Comment:
@webimpress I agree, it also requires to refactor all `getMock()` calls.
I changed the mock creation for simple mocks. I prefer to use the Mock Builder when I need to disable the constructor, it's more declarative and readable.